CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+4.54.1
+==========
+### New Features
+- The client-side telemetry API (`ITelemetryClient`) is now generally available. See [3784](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3784).  
+- Logging additional telemetry data: exception information, access token type, token refresh on value, assertion type, authority, scopes, cache level used. See [3784](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3784).  
```

Comment:
Probably dont need to mention specific data for telemetry. Customers dont need to know since it is primarily for internal use